examples/research_projects/fsner/README.md

```diff
@@ -0,0 +1,86 @@
+# FSNER
+
+Implemented by [sayef](https://huggingface.co/sayef).
+
+## Overview
+
+The FSNER model was proposed in [Example-Based Named Entity Recognition](https://arxiv.org/abs/2008.10570) by Morteza Ziyadi, Yuting Sun, Abhishek Goswami, Jade Huang, Weizhu Chen. To identify entity spans in a new domain, it uses a train-free few-shot learning approach inspired by question-answering.
+
+
+
+## Abstract
+----
+> We present a novel approach to named entity recognition (NER) in the presence of scarce data that we call example-based NER. Our train-free few-shot learning approach takes inspiration from question-answering to identify entity spans in a new and unseen domain. In comparison with the current state-of-the-art, the proposed method performs significantly better, especially when using a low number of support examples.
+
+
+
+## Model Training Details
+-----
+
+| identifier        | epochs           | datasets  |
+| ---------- |:----------:| :-----:|
+| sayef/fsner-bert-base-uncased      | 10 | ontonotes5, conll2003, wnut2017, and fin (Alvarado et al.). |
```

Comment:
Perhaps you can make sayef/fsner-bert-base-uncased a hyperlink to the model on the hub

Reply:
I have added a hyperlink to the model on the hub.